aea/cli/utils/click_utils.py

```diff
@@ -208,32 +208,57 @@ def convert(self, value: str, param: Any, ctx: click.Context) -> str:  # type: i
 def registry_flag(mark_default: bool = True) -> Callable:
     """Choice of one flag between: '--local/--remote'."""
 
-    cli_config = get_or_create_cli_config()
-    default_registry = cast(
-        str, cli_config.get(REGISTRY_CONFIG_KEY, {}).get("default", REGISTRY_LOCAL)
+    default_registry = (
+        get_or_create_cli_config()
+        .get("registry_config", {})
+        .get("default", REGISTRY_LOCAL)
     )
 
     def wrapper(f: Callable) -> Callable:
         f = option(
             "--local",
             "registry",
             flag_value=REGISTRY_LOCAL,
-            help="For pushing items to local folder.",
+            help="To use a local registry.",
             default=(REGISTRY_LOCAL == default_registry) and mark_default,
         )(f)
         f = option(
-            "--ipfs",
+            "--remote",
             "registry",
-            flag_value=REGISTRY_IPFS,
-            help="For pushing items to ipfs node.",
-            default=(REGISTRY_IPFS == default_registry) and mark_default,
+            flag_value=REGISTRY_REMOTE,
+            help="To use a remote registry.",
+            default=(REGISTRY_REMOTE == default_registry) and mark_default,
```

Comment:
Now we have only 2 main types of registries, local and remote. Using local flag will use the local packages folder. Remote have 2 subtypes ipfs and http one of which can be set as default in global config when initialising aea.